README.md

```diff
@@ -1,6 +1,19 @@
 # htbhf-acceptance-tests
 Acceptance tests for Apply for Healthy Start application
 
+These tests are downloaded and run by the htbhf-applicant-web-ui project 
+(which specifies the version of these tests to run in `test_versions.properties`). 
+The acceptance tests are run as part of the CI build for htbhf-applicant-web-ui and the compatibility tests are run during the CD pipeline.
+
+## Running the acceptance tests against a local instance of htbhf-applicant-web-ui
+When developing new features you can run against your local copy of htbhf-applicant-web-ui by running the 
+`run_tests_with_local_feature_toggles.sh` script. The tests require that applicant-web-ui is running (`npm start`), 
+and that the session details app is also running (`npm run test:session-details`). 
+All other endpoints (claimant service and os-places) are mocked out by the tests.
+
+When pushing your changes to GitHub, tell circleci which branch of htbhf-applicant-web-ui it should test against 
```

Comment:
Isn't this dangerous? At what point should this be removed so that when the merge happens, the tests pass. Would be worth a sentence or two to explain that scenario.

Reply:
Hopefully the updates make it clear?